### ISSUE TITLE
speed issues on windows

### ISSUE THREAD

**Issue by sanderboer** [ananthakumaran/tide#27]
Hi,

I know - windows- *sigh*, sorry. We all kneel before that mountain sometimes.

Now, my issue is that codecompletion on an angular2 and three js boilerplate project is a good 6 seconds before company comes back. Backend time-outs are rife. Flycheck is annoying because it lags my cursor movements.
Now this is on a surface pro, the first one and it is no speedmonster, but the difference in speed with vscode is intense. vscode just flies.

So, some tips on how to improve speed would be greatly appreciated.

Alternatively, I would like to propose that linting and codecompletion is done by command, a shortcut key. This way  I am in control of the wait and I am pushed to just get it right.
Usually I use codecompletion as a way to quickbrowse a library, maybe there is another way to quickly browse a lib ?

tips appreciated,
Sander

**Comment by ananthakumaran** [comment]
AFAIK, vscode uses the same backend as tide. So things should be as fast as vscode if done properly. I have added info on how to enable the log file on [readme](https://github.com/ananthakumaran/tide#debugging). Have a look at the log file. It provides timing information about each and every operation.

**Comment by sanderboer** [comment]
I think this issue can be closed.
I noticed erratic behavior also with npm and I figured the d:\ drive on the surface is failing, it's a microsd. Since there is (relatively)  a lot of disk io with node, performance would be impacted severely by a failing drive. VScode is installed on c:, hence the performance diff.
Moved my project to c:\ and all is well, (insofar possible on win, it's still a massive pitb).
